doc/source/upgrading_guide.rst

```diff
@@ -34,6 +34,21 @@ of incompatible changes that could cause automatic upgrade to fail:
    With partitioned tables that use the TokuDB or MyRocks storage
    engine, the upgrade only works with native partitioning.
 
+.. note::
+
+    :ref:`encrypting-threads` is an **EXPERIMENTAL** feature.
+    
+    Before encryption is set to "ON", the user should consult the
+    INFORMATION_SCHEMA_ENCRYPTION table to verify which tables are excluded from
+    the encryption. 
+
+    If the user wants to exclude a table from encryption, the user should run
+    ``ALTER TABLE ... ENCRYPTION='N'``. In the case of ``ENCRYPTION='Y'``, the
+    table remains encrypted and a table that is not encrypted remains not
+    encrypted. 
+
+    During an upgrade, all non-encrypted tables are set to ``ENCRYPTION='N'``.
+
```

Comment:
Here we also need to distinguish between explicit ENCRYPTION='N' and implicit ENCRYPTION='N'

Reply:
Rewritten.